pkg/dao/repository_test.go

```diff
@@ -352,12 +388,19 @@ func (suite *RepositorySuite) TestUpdateBlank() {
 	name := "Updated"
 	url := "http://someUrl.com"
 	blank := ""
-	orgID := "900023"
+	orgID := orgIdTest
+
+	repo := repoPublicTest.DeepCopy()
+	err = tx.Create(&repo).Error
+	require.NoError(t, err)
+
+	repoConfig := repoConfigTest1.DeepCopy()
+	repoConfig.RepositoryUUID = repo.UUID
+	err = tx.Create(&repoConfig).Error
+	require.NoError(t, err)
```

Comment:
add records by hand to have better control of the records to be recovered later.